resources/views/components/nav.blade.php

```diff
@@ -13,8 +13,24 @@
         <div id="navbar" class="navbar-collapse collapse">
             <ul class="nav navbar-nav navbar-right">
                 @if (Auth::user())
-                    <li><a class="nav-item nav-link" href="/import">Import</a></li>
-                    <li><a class="nav-item nav-link" href="/logout">Logout</a></li>
+                    <li class="{{ (app('request')->input('type') === get_import_type_turbovote()) ? 'active' : '' }}">
```

Comment:
Totally subjective, but you can use the `request()` helper here as a shortcut to `app('request')`.